app/javascript/app/components/home/ndc-pledge/ndc-pledge-styles.scss

```diff
@@ -16,25 +16,41 @@
   @include columns();
 
   @media #{$tablet-landscape} {
-    @include columns(6);
+    @include columns($columns: (6, 5, 1));
   }
 
+  @media #{$desktop} {
+    @include columns($columns: (6, 4, 2));
+  }
+
+  background-color: $porcelain;
+
   .ndcImage {
-    max-width: 100%;
     object-fit: cover;
```

Comment:
you do not need both `object-fit: cover` and `background-size: cover`, you can safely remove object-fit